### DISCUSSION TITLE
Go back to the original size

### DISCUSSION THREAD

**Discussion by so27** [mrkai77/Loop#654] [Q&A]
Hello, 

is it possible that if I press the 🌐 + Space keys and press them again in the window, that the window goes back to its original size? Or is there a key combination that resets the window to its normal size, as if I move it with the mouse?

**Comment by mrkai77** [comment]
Yes, you can add your own keybind in Loop's settings called initial frame to achieve this :)

**Reply by so27** [reply]
Perfect. Now it works perfectly. Thank you very much.
